Appraisals

```diff
@@ -66,17 +66,33 @@ appraise 'rails-4.2' do
 end
 
 # Rails 5+ supports only Ruby 2.1+
```

Comment:
I think this should be [`Ruby 2.2.2+`](https://github.com/rails/rails/commit/32f7491808d2c4e097ed7b3ee875e4d1cea8c442#diff-73ac7236f7a8247a9cd61d338ef49cda).